lib/rbs/type_alias_dependency.rb

```diff
@@ -74,6 +76,8 @@ def direct_dependency(type, result = Set[])
       # Append type name if the type is an alias
       result << type.name if type.is_a?(RBS::Types::Alias)
 
+      return result if SAFELIST.include?(type.class)
```

Comment:
Please write a case analysis using `case-when` here, instead of having `SAFELIST`.